course_info/templates/course_info/partials/details.html

```diff
@@ -0,0 +1,211 @@
+                            <ng-pluralize count="dc.courseInstance.members"
+                                          when="{'0': 'There are no people in the course.',
+                                'one': 'There is one person in the course.',
+                                'other': 'There are {} people in the course.'}">
+                            </ng-pluralize>
+                        </a>
+                    </div>
+                </div>
+            </li>
+
+            <li class="list-group-item">
+                <div class="row">
+                    <div class="col-md-2">
+                        <strong>Location: </strong>
+                    </div>
+                    <div class="col-md-10">{{dc.courseInstance.location}}</div>
+                </div>
+            </li>
+
+            <li class="list-group-item">
+                <div class="row">
+                    <div class="col-md-2">
+                        <strong>Meeting Time: </strong>
+                    </div>
+                    <div class="col-md-10">{{dc.courseInstance.meeting_time}}</div>
+                </div>
+            </li>
+
+            <li class="list-group-item">
+                <div class="row">
+                    <div class="col-md-2">
+                        <strong>Departments: </strong>
+                    </div>
+                    <div class="col-md-10" ng-repeat="dept in dc.courseInstance.departments">
+                         <div class="list-group" ng-if="dc.courseInstance.departments.length > 0">
+                            <div class="list-group-item list-group-sub-item" ng-repeat="dept in dc.courseInstance.departments">
+                                {{dept.name}}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </li>
+
+            <li class="list-group-item">
+                <div class="row">
+                    <div class="col-md-2">
+                        <strong>Course Groups: </strong>
+                    </div>
+                    <div class="col-md-10">
+                        <div class="list-group">
+                            <div class="list-group-item list-group-sub-item" ng-repeat="group in dc.courseInstance.course_groups">
+                                {{group.name}}
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </li>
+
+            <li class="list-group-item">
+                <div class="row">
+                    <div class="col-md-2">
+                        <strong>Associated Official and Unofficial Site(s): </strong>
+                    </div>
+                    <div class="col-md-10">
+                        <div class="list-group">
+                            <div class="list-group-item list-group-sub-item" ng-repeat="site in dc.courseInstance.sites">
```

Comment:
At small font sizes in Chrome we get some funky display behavior here... might be worth just 'bugging' it and not troubleshooting it as part of this PR, though.

<img width="439" alt="screen shot 2016-03-18 at 11 00 39 am" src="https://cloud.githubusercontent.com/assets/8975317/13881794/cdf76bac-ecf8-11e5-8931-4c7d27e9b55a.png">